### PR TITLE
HPCC-21152 Active WUs not attaching to correct queue

### PR DESCRIPTION
**Pull request by GordonSmith**
Removed some erroneous debug statements

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq please review (and test)

**Comment by miguelvazq** [comment]
Code looks good.

Testing: It would be ideal to add the ServerName : queuename
Line #205 ESPQueue.ts

`return this.ServerName + (this.ClusterName ? " - " + this.ClusterName : "");`

**Comment by GordonSmith** [comment]
@miguelvazq ready for review

File: esp/src/src/ESPQueue.ts [hunk 1]
```diff
@@ -347,22 +349,22 @@ export function isInstanceOfQueue(obj) {
     return obj && obj.isInstanceOf && obj.isInstanceOf(Queue);
 }
 
-export function GetTargetCluster(name) {
+export function GetTargetCluster(name, createIfMissing = false) {
```
Comment (miguelvazq) [inline esp/src/src/ESPQueue.ts]: This syntax seems odd

File: esp/src/src/ESPQueue.ts [hunk 2]
```diff
@@ -347,22 +349,22 @@ export function isInstanceOfQueue(obj) {
...
-    if (!retVal) {
+    if (!retVal && createIfMissing) {
         retVal = new TargetCluster(id);
         store.put(retVal);
     }
     return retVal;
 }
 
-export function GetServerJobQueue(name) {
+export function GetServerJobQueue(name, createIfMissing = false) {
```
Comment (miguelvazq) [inline esp/src/src/ESPQueue.ts]: This syntax seems odd

**Review by miguelvazq — COMMENTED** [review]
Minor comment.

File: esp/src/src/ESPQueue.ts [hunk 3]
```diff
@@ -202,7 +204,7 @@ var TargetCluster = declare([Queue], {
     },
 
     getDisplayName: function () {
-        return this.ClusterName;
+        return this.ServerName + (this.ClusterName ? " - " + this.ClusterName : "");
```
Comment (miguelvazq) [inline esp/src/src/ESPQueue.ts]: return this.ServerType + (this.ClusterName ? " - " + this.ClusterName : "");

**Review by miguelvazq — CHANGES_REQUESTED** [review]

File: esp/src/src/ESPQueue.ts (hunk 2 above)
Comment (miguelvazq) [reply esp/src/src/ESPQueue.ts]: As discussed this seems fine.

**Comment by GordonSmith** [comment]
@miguelvazq change made as requested.

File: esp/src/src/ESPQueue.ts (hunk 1 above)
Comment (miguelvazq) [reply esp/src/src/ESPQueue.ts]: As discussed this seems fine.

**Comment by miguelvazq** [comment]
Tested and looks good.

**Review by miguelvazq — APPROVED** [review]

**Comment by miguelvazq** [comment]
@richardkchapman Ready for merge.

**Pull request merged**